test/image/mocks/zindex_basic.json

```diff
@@ -0,0 +1,53 @@
+{
+    "data": [
+        {
+            "x": [2, 3, 4, 5],
+            "y": [4, 3, 9, 5],
+            "name": "blue",
+            "type": "scatter",
+            "marker": {"size": 20},
+            "line": {"width": 6},
+            "zindex": 5
```

Comment:
Let's improve this mock by using some negative `zindex` values as well as overlaying few more bars too.